src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        Color color = new ColorSupplier().getRandomColor();
+        int number = random.nextInt(101);
```

Comment:
Magic numbers should be avoided and replaced with named constants for better code readability. Consider creating a constant for the number 101, giving it a meaningful name that explains what it represents.

Reply:
fix comment

Reply:
not fixed